go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	k8s.io/client-go v0.22.3
 	k8s.io/component-helpers v0.22.3
 	k8s.io/klog v1.0.0
+	k8s.io/klog/v2 v2.9.0
```

Comment:
we should not mix klog v1 and klog v2 in one project, maybe we can use klog v1 in this pull request, and upgrade klog v1 to klog v2 in new pull request.

Reply:
fixed